mitosheet/mitosheet/tests/steps_performers/column_steps/test_split_text_to_columns.py

```diff
@@ -0,0 +1,148 @@
+from mitosheet.tests.test_utils import create_mito_wrapper_dfs
+
+
+SPLIT_TEXT_TO_COLUMNS_TESTS = [
+    (
+        pd.DataFrame({'A': ['1,2-3\t4 5/6']}),
+        'A',
+        [','],
+        pd.DataFrame({'A': ['1,2-3\t4 5/6'], 'A-split-0-1': ['1'], 'A-split-1-1': ['2-3\t4 5/6']})
+    ),
+    (
+        pd.DataFrame({'A': ['1,2-3\t4 5/6']}),
+        'A',
+        ['-'],
+        pd.DataFrame({'A': ['1,2-3\t4 5/6'], 'A-split-0-1': ['1,2'], 'A-split-1-1': ['3\t4 5/6']})
+    ),
+    (
+        pd.DataFrame({'A': ['1,2-3\t4 5/6']}),
+        'A',
+        ['\t'],
+        pd.DataFrame({'A': ['1,2-3\t4 5/6'], 'A-split-0-1': ['1,2-3'], 'A-split-1-1': ['4 5/6']})
+    ),
+    (
+        pd.DataFrame({'A': ['1,2-3\t4 5/6']}),
+        'A',
+        [' '],
+        pd.DataFrame({'A': ['1,2-3\t4 5/6'], 'A-split-0-1': ['1,2-3\t4'], 'A-split-1-1': ['5/6']})
+    ),
+    (
+        pd.DataFrame({'A': ['1,2-3\t4 5/6']}),
+        'A',
+        ['/'],
+        pd.DataFrame({'A': ['1,2-3\t4 5/6'], 'A-split-0-1': ['1,2-3\t4 5'], 'A-split-1-1': ['6']})
+    ),
+    (
+        pd.DataFrame({'A': ['1,2-3\t4 5/6']}),
+        'A',
+        ['\t',','],
+        pd.DataFrame({'A': ['1,2-3\t4 5/6'], 'A-split-0-1': ['1'], 'A-split-1-1': ['2-3'], 'A-split-2-1': ['4 5/6']})
+    ),
+    (
+        pd.DataFrame({'A': ['1,2-3\t4 5/6']}),
+        'A',
+        [',','/'],
+        pd.DataFrame({'A': ['1,2-3\t4 5/6'], 'A-split-0-1': ['1'], 'A-split-1-1': ['2-3\t4 5'], 'A-split-2-1': ['6']})
+    ),
+    (
+        pd.DataFrame({'A': ['1,2-3\t4 5/6']}),
+        'A',
+        [',', '/', '\t', ' ', '-'],
+        pd.DataFrame({'A': ['1,2-3\t4 5/6'], 'A-split-0-1': ['1'], 'A-split-1-1': ['2'], 'A-split-2-1': ['3'], 'A-split-3-1': ['4'], 'A-split-4-1': ['5'], 'A-split-5-1': ['6']})
+    ),
+    (
+        pd.DataFrame({'A': ["12'34'56"]}),
+        'A',
+        ["'"],
+        pd.DataFrame({'A': ["12'34'56"], 'A-split-0-1': ['12'], 'A-split-1-1': ['34'], 'A-split-2-1': ['56']})
+    ),
+    (
+        pd.DataFrame({'A': ['12"34"56']}),
+        'A',
+        ['"'],
+        pd.DataFrame({'A': ['12"34"56'], 'A-split-0-1': ['12'], 'A-split-1-1': ['34'], 'A-split-2-1': ['56']})
+    ),
+    (
+        pd.DataFrame({'A': [1.23, 1.23]}),
+        'A',
+        ['.'],
+        pd.DataFrame({'A': [1.23, 1.23], 'A-split-0-1': ['1', '1'], 'A-split-1-1': ['23', '23']})
+    ),
+    (
+        pd.DataFrame({'A': pd.to_datetime(['1/2/22', '4/06/99'])}),
+        'A',
+        [' ', '-'],
+        pd.DataFrame({'A': pd.to_datetime(['1/2/22', '4/06/99']), 'A-split-0-1': ['2022', '1999'], 'A-split-1-1': ['01', '04'], 'A-split-2-1': ['02', '06'], 'A-split-3-1': ['00:00:00', '00:00:00']})
+    ),
+    (
+        pd.DataFrame({'A': [pd.to_datetime('3/07/21') - pd.to_datetime('1/12/22')]}),
+        'A',
+        [' '],
+        pd.DataFrame({'A': [pd.to_datetime('3/07/21') - pd.to_datetime('1/12/22')], 'A-split-0-1': ['-311'], 'A-split-1-1': ['days'], 'A-split-2-1': ['+00:00:00']})
+    ),
```

Comment:
Missing tests for timedelta.

Reply:
This is a timedelta test. I'm just creating a timedelta by subtracting the two datetimes.